.github/workflows/build_and_release.yml

```diff
@@ -0,0 +1,28 @@
+name: Build And Deploy App to Firebase Distribution
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
We usually have two separate deployment workflows for staging/dev and production. For this project, I think we should deploy the staging build every time a PR is merged to `develop`.